### DISCUSSION TITLE
Is there a way to only generate the depth map in Auto1111?

### DISCUSSION THREAD

**Discussion by SebSaga** [thygate/stable-diffusion-webui-depthmap-script#67] [Q&A] [answered]
When I run the script, it generates a depth map after doing an initial im2img. Is there any way to skip the initial img2img and just have the script output the depth map by itself?

I only need the depth map, so it seems to take twice as long as it should for what I need. Any help would be greatly appreciated. Thanks :)

**Comment by thygate** [comment, marked as answer]
Do you want to run it on existing images, is that what you're asking ?

You can use img2img with denoising strength 0 and set size to match manually, or you can use the depth tab .. It's all in the readme under Usage ..

**Reply by SebSaga** [reply]
Ah yes I see I'm so sorry, I confused this extension for this one: https://github.com/Extraltodeus/depthmap2mask

This extension is exactly what I need. I installed it, and it's working perfectly. Thank you so much!
